imap_processing/hi/l1a/histogram.py

```diff
@@ -108,6 +108,13 @@ def allocate_histogram_dataset(num_packets: int) -> xr.Dataset:
         dims=["angle"],
         attrs=attr_mgr.get_variable_attributes("hi_hist_angle"),
     )
+    coords["angle_label"] = xr.DataArray(
```

Comment:
Label variables should not be added to the coordinates list. They are just variables.

Reply:
This LABL_PTR_i confused me a lot. Changes in this file and XML file is related to that. This is something SPDF and ISTP caused. When your data has great than 1D array, it requires to have label for each `DEPEND_1` and onward. And we need to create xr.DataArray for those labels and need to add those to coordinate, otherwise `cdflib` will throw error.
```
FAILED imap_processing/tests/hi/test_l1a.py::test_app_hist_decom - cdflib.xarray.xarray_to_cdf.ISTPError: ISTP Compliance Warning: variable qual_ab listed angle_label as its LABL_PTR_1.  However, it was not found in the dataset.
```

Reply:
```suggestion
    data_vars["angle_label"] = xr.DataArray(
```
I think that is what @subagonsouth  is suggesting. cdflib fails with that as well? Coordinates should be optional, so this might be an issue to raise with cdflib.

Reply:
moving this to data variable. I will refactor other instrument's label in new PR. I do want to point out that other past mission did store their coordinate label in coordinate as well. I wonder if they did that to be able to select and filter data easier later on. That's something we should think about. I will check with Andriy first before I refactor about this too.